frame/composable-support/src/validation.rs

```diff
@@ -132,6 +218,9 @@ mod test {
 	type CheckARangeTag = (ValidARange, Valid);
 	type CheckBRangeTag = (ValidBRange, Valid);
 	type CheckABRangeTag = (ValidARange, (ValidBRange, Valid));
+	// type ManyValidatorsTagsNested = (ValidARange, (ValidBRange, (Invalid, Valid)));
```

Comment:
@hussein-aitlahcen suggested that nested is option to choose. will it still work?

Reply:
I think this will still work as long as we keep the impl for (T, U) & Valid, such that we can do (T, Valid), (T, (U, Valid)), (T, (U, (V, Valid)))... So perhaps we should uncomment?